src/shared/prosemirror-plugins/placeholder.ts

```diff
@@ -0,0 +1,51 @@
+import { Node } from "prosemirror-model";
+import { Plugin } from "prosemirror-state";
+import { Decoration, DecorationSet } from "prosemirror-view";
+
+function showPlaceholder(content: Node) {
+    const { firstChild } = content;
+    const { name } = firstChild.type;
+    const allowPlaceholder =
+        name === "paragraph" ||
+        name === "heading" ||
+        firstChild.attrs.params === "markdown";
```

Comment:
This causes the placeholder to show in rich-text mode when you load the following markdown:

~~~markdown
```markdown
```
~~~

Reply:
Ah, I just realized I never addressed this issue. Looking now.